src/codegen/bindgen-lib.ts

```diff
@@ -196,6 +200,23 @@ export namespace t {
   export const ByteString = builtinType<string>()("ByteString");
   /**
    * DOMString but encoded as `[]const u8`
+   *
+   * @example
```

Comment:
i think simple examples like these aren't worth their line count, especially when my laptop screen can only fit 70 lines of code at once. this is why i try to write extremely concise comments.

Reply:
It makes it appear nicely when hovering over the symbol.
<img width="485" alt="image" src="https://github.com/user-attachments/assets/f862dd2c-abb5-4848-83d7-9246040caf0e" />

If this was ts-only, VSCode would still perform syntax highlighting.

Reply:
Ah the hover doesn't appear in screenshots :(